robottelo/helpers.py

```diff
@@ -143,3 +143,53 @@ def read_data_file(filename):
     absolute_file_path = get_data_file(filename)
     with open(absolute_file_path, 'r') as file_contents:
         return file_contents.read()
+
+
+def install_katello_ca(hostname=None):
+        """Downloads and installs katello-ca rpm
+
+        :param str hostname: Hostname or IP address of the remote host. If
+        ``None`` the hostname will be get from ``main.server.hostname`` config.
+        :return: None.
+        :raises AssertionError: If katello-ca wasn't installed.
+        """
+        ssh.command(
+            u'rpm -Uvh {0}'.format(settings.server.get_cert_rpm_url()),
+            hostname
+        )
+        # Not checking the return_code here, as rpm could be installed before
+        # and installation may fail
+        result = ssh.command(
+            u'rpm -q katello-ca-consumer-{0}'
+            .format(settings.server.hostname),
+            hostname
+        )
+        # Checking the return_code here to verify katello-ca rpm is actually
+        # present in the system
+        if result.return_code != 0:
+            raise AssertionError('Failed to install the katello-ca rpm')
+
+
+def remove_katello_ca(hostname=None):
+        """Removes katello-ca rpm
+
+        :param str hostname: Hostname or IP address of the remote host. If
+        ``None`` the hostname will be get from ``main.server.hostname`` config.
+        :return: None.
+        :raises AssertionError: If katello-ca wasn't removed.
+        """
+        ssh.command(
```

Comment:
Maybe I got something wrong, but later at that disccussion @omaciel and @sthirugn agreed that it's save and enough to simply delete `rpm` and that's it, didn't they?

As per their comments:
> It doesn't hurt us at all.

 and
> @omaciel is right. Since we are not getting sat tools in the sat host itself. But we do it from the satellite application which is configured with cdn info already. Registering the satellite host to itself should not be a problem for the sync tests.

Although I agree it's better to double check. Will check some random sync & longrun tests.